nupic/frameworks/opf/clamodel.py

```diff
@@ -814,6 +832,12 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
       likelihoodsDict = CLAModel._removeUnlikelyPredictions(
           likelihoodsDict, minLikelihoodThreshold, maxPredictionsPerStep)
 
+      # calculate likelihood for each bucket
+      bucketLikelihood = {}
+      for k in likelihoodsDict.keys():
+        bucketLikelihood[self._classifierInputEncoder.getBucketIndices(k)[0]] = \
```

Comment:
Please use parenthesis instead of `\` to split code among multiple lines, here and everywhere lese.